src/iosrv/drpc_client.c

```diff
@@ -113,14 +113,13 @@ notify_bio_error(bool unmap, bool update, int tgt_id)
 	if (rc != 0)
 		return rc;
 
-	if (unmap)
+	/* TODO: add checksum error */
```

Comment:
why not add a bioerr_req.cserr as a part of this patch? this is just a boolean type that will get displayed if the value is true for error type. For example - in the below output, you can tell it is a write error since writeErr is true (and all the other types aren't displayed because they are false)
ERROR: I/O server instance 0 (target 0) has detected blob I/O error! writeErr:true drpcListenerSock:"/tmp/daos_io_server_71020.sock" uri:“ofi+sockets://10.7.1.8:31416“

Reply:
There is a follow-up patch planned to complete the CSUM error logging.  The follow-on will add support for logging checksum mismatches for keys.  This key-checking capability is under development (by Ryon), and is scheduled to land soon.  If okay, I'd like to wait until the follow-on patch to implement the error notification for CSUM errors.